mod_reforged/hooks/config/strings.nut

```diff
@@ -297,7 +297,8 @@ local vanillaDescriptions = [
 	 		Effects = [{
  				Type = ::UPD.EffectType.Active,
  				Description = [
-					"Unlocks the [Taunt|Skill+taunt] skill which makes the targeted opponent take offensive actions instead of defensive ones, and attack the taunting character over another, potentially more vulnerable one."
+					"Unlocks the [Taunt|Skill+taunt] skill which makes the targeted opponent take offensive actions instead of defensive ones, and attack the taunting character over another, potentially more vulnerable one.",
+					"When used on an adjacent target, reduce its [Melee Defense|Concept.MeleeDefense] and [Ranged Defense|Concept.RangeDefense] by " + ::MSU.Text.colorGreen("20%") + " of the users [Resolve|Concept.Resolve]."
```

Comment:
- `user's`
- also `reduces` instead of `reduce` would be more appropriate to further describe the effect of the skill like the first paragraph which says it `makes` the target do something.